.github/workflows/update-search.yml

```
@@ -35,21 +35,37 @@ jobs:
-          # Fetch latest changes from origin/main
-          git fetch origin main
-          
           # Add changes
           git add docs/assets/js/search_db.json
           
           if git diff --staged --quiet; then
             echo "No changes to commit"
           else
+            # Fetch and pull latest changes before committing
+            git pull origin main --rebase
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)